### DISCUSSION TITLE
Safari iOS 13.7 homepage route cannot be loaded, and the error page is also blank.

### DISCUSSION THREAD

**Discussion by dxrvip** [vercel/next.js#55360] [App Router]
"@headlessui/react": "^1.7.16",
    "@heroicons/react": "^2.0.18",
    "@tailwindcss/forms": "^0.5.4",
    "@types/node": "^18.6.3",
    "@types/react": "^18.0.15",
    "@types/react-dom": "^18.0.6",
    "classnames": "^2.3.1",
    "clientjs": "^0.2.1",
    "date-fns": "^2.28.0",
    "moment": "^2.29.4",
    "next": "latest",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "swr": "^2.2.0",
    "typescript": "^4.7.4"

IOS Safari I'm using the latest version of nextjs
flash blank

**Comment by icyJoseph** [comment]
Got any logs from your application? You can inspect iOS from a Mac.

**Reply by dxrvip** [reply]
Error: Cancel rendering route

**Reply by dxrvip** [reply]
This error only occurs on the browser of the mobile phone iOS 13.7 system

**Reply by dxrvip** [reply]
TypeError: undefined is not an object (evaluating 'beforeHydrationMeasure.startTime')

**Reply by icyJoseph** [reply]
It has to be this then https://github.com/vercel/next.js/issues/54389

**Reply by icyJoseph** [reply]
Apparently the latest Canary should have a fix:

- https://github.com/vercel/next.js/issues/55084#issuecomment-1712455398
